### PR TITLE
add continuous actions option in maddpg

### PR DESCRIPTION
**Pull request by rical730**

### CODE REVIEW

**Comment by rical730** [comment]
fix https://github.com/PaddlePaddle/PARL/issues/804

File: benchmark/torch/maddpg/train.py [hunk 1]
```diff
@@ -19,15 +19,15 @@
 from simple_model import MAModel
 from simple_agent import MAAgent
 from parl.algorithms import MADDPG
-from parl.env.multiagent_simple_env import MAenv
+from parl.env.pettingzoo_mpe import MAenv_v2
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/maddpg/train.py]: Can we rename the env path in PARL:
`from parl.env.multiagent_env import env ` ?
The user does not care the env version.

Comment (rical730) [reply benchmark/torch/maddpg/train.py]: What about renaming with `from parl.env.multiagent_env import MAenv`. 
Using `import env` may be inconvenient to set the variable name `env` when developing train.py.

**Review by TomorrowIsAnOtherDay — COMMENTED** [review]
LGTM.

**Review by TomorrowIsAnOtherDay — DISMISSED** [review]

File: parl/env/multiagent_simple_env.py [hunk 2]
```diff
@@ -67,6 +68,11 @@ def get_shape(self, input_space):
                     '[Error] shape is {}, not Box or Discrete or MultiDiscrete'
                     .format(input_space.shape))
                 raise NotImplementedError
+
+    logger.warning(
+        'the `MAenv` from `parl.env.multiagent_simple_env` is deprecated since parl 2.0.4 and will be removed in parl 3.0. \n \
+        We recomend you to use `from parl.env.multiagent_env import MAenv` instead, which provides arguments for continuous_actions.'
```
Comment (TomorrowIsAnOtherDay) [inline parl/env/multiagent_simple_env.py]: provides arguments for continuous_actions. ->
supports continuous control.

Comment (rical730) [reply parl/env/multiagent_simple_env.py]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]

**Pull request merged**